### PR TITLE
Information about data calculations using the STH component

### PR DESCRIPTION
**Pull request by gtorodelvalle**

### CODE REVIEW

**Comment by fgalan** [comment]
LGTM

File: docs/topics/data_calculation.md [hunk 1]
```diff
@@ -116,4 +116,17 @@ For those cases when the data values that want to be calculated are statistical
...
 to retrieve statistical calculations over the stored data.
 
-// @GERMAN: SOME INFORMATION ABOUT HOW TO GET STATISTICS CALCULATED BY THE STH WOULD BE NEEDED
\ No newline at end of file
+As thoroughly detailed in the [STH documentation at ReadTheDocs](http://fiware-sth-comet.readthedocs.io/en/latest/index.html), the STH is able to calculate statistics about the evolution in time of certain entity attributes. To do it, there are 2 ways to notify this evolution in time to the component:
+
+1. Via the [Cygnus](https://github.com/telefonicaid/fiware-cygnus) component and subscribing it to the Context Broker instance.
+2. Via the [STH](https://github.com/telefonicaid/fiware-sth-comet) component itself and directly subscribing it to the Context Broker instance.
+
+Once the STH receives the notifications of entity attribute value changes, it is able to calculate and provide information about the mean, standard deviation, variance, maximum and minimum values (for numerical values) as well as the number of ocurrences (for textual values) for distinct resolutions or time frames (such as months, days, hours, minutes and seconds).
```
Comment (dmoranj) [inline docs/topics/data_calculation.md]: This paragraph may be easier to read listing the options with bullets. Take into account that people usually make diagonal reads on this topics, so we better highlight the available options.

File: docs/topics/data_calculation.md [hunk 2]
```diff
@@ -116,4 +116,17 @@ For those cases when the data values that want to be calculated are statistical
...
+
+Regarding the previous example, the STH is able to provide statistical information such as:
+
+1. Which has been the maximum `weight` (attribute) values of the `WasteTank8` (entity) last year with a resolution of months?
```
Comment (dmoranj) [inline docs/topics/data_calculation.md]: Can you provide an example of the query that would calculate that? The same goes for the following points.

File: docs/topics/data_calculation.md [hunk 3]
```diff
@@ -116,4 +116,17 @@ For those cases when the data values that want to be calculated are statistical
...
+2. Which has been the mean `level` (attribute) values of the `WasteTank8` (entity) last week with a resolution of hours?
+3. Which has been the standard deviation of the `density` (attribute) values of the `WasteTank8` the last 3 days with a resolution of hours or even minutes?
+
+As already mentioned, for futher information, please visit the [STH documentation at ReadTheDocs](http://fiware-sth-comet.readthedocs.io/en/latest/index.html).
```
Comment (dmoranj) [inline docs/topics/data_calculation.md]: Mentioning it twice seems a bit strange. I think we should better remove the first reference and let the last one be the detailed reference of the document.

File: docs/topics/data_calculation.md (hunk 1 above)
Comment (gtorodelvalle) [inline docs/topics/data_calculation.md]: Agreed! ;) On it...

File: docs/topics/data_calculation.md (hunk 2 above)
Comment (gtorodelvalle) [inline docs/topics/data_calculation.md]: Agreed! ;)

File: docs/topics/data_calculation.md (hunk 3 above)
Comment (gtorodelvalle) [inline docs/topics/data_calculation.md]: Agreed! ;)

File: docs/topics/data_calculation.md [hunk 4]
```diff
@@ -116,4 +116,34 @@ For those cases when the data values that want to be calculated are statistical
...
+
+* Months
+* Days
+* Hours
+* Minutes
+* Seconds
+
+Regarding the previous example, the STH is able to provide statistical information such as:
+
+1. Which has been the maximum `weight` (attribute) values of the `WasteTank8` (entity) last year with a resolution of months?, sending a GET request to the STH component such as the next one: `http://<sth-component>:<sth-port>/STH/v1/contextEntities/type/Entity/id/WasteTank8/attributes/weight?aggrMethod=max&aggrPeriod=month&dateFrom=2015-01-01T00:00:00&dateTo=2015-12-31T23:59:59`
```
Comment (dmoranj) [inline docs/topics/data_calculation.md]: The result of writing the query in the same sentence is strange. Can you move it to a code block below?

**Pull request merged**